deploy/variables.tf

```diff
@@ -4,6 +4,12 @@ variable "project_id" {
   nullable    = false
 }
 
+variable "project_name" {
+  description = "Oystehr project name"
+  type        = string
+  nullable    = false
+}
```

Comment:
variables of the format `#{var/PROJECT_NAME}` are stored in the config files that live in `packages/zambda/.env`. they aren't terraform variables, they're... ottehr variables 😅

Reply:
c870c31